test/controllers/articles_controller_test.rb

```diff
@@ -0,0 +1,57 @@
+require "test_helper"
+
+class ArticlesControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @article = articles(:one)
+  end
+
+  test "should get index" do
+    get articles_url
+
+    assert_response :success
+  end
+
+  test "should get new" do
+    get new_article_url
+
+    assert_response :success
+  end
+
+  test "should create article" do
+    assert_difference("Article.count") do
+      post articles_url,
+           params: { article: { article_body: @article.article_body, article_title: @article.article_title,
+                                creator_name: @article.creator_name } }
+    end
+
+    assert_redirected_to article_url(Article.last)
+  end
+
+  test "should show article" do
+    get article_url(@article)
+
+    assert_response :success
+  end
+
+  test "should get edit" do
+    get edit_article_url(@article)
+
+    assert_response :success
+  end
+
+  test "should update article" do
+    patch article_url(@article),
+          params: { article: { article_body: @article.article_body, article_title: @article.article_title,
+                               creator_name: @article.creator_name } }
+
+    assert_redirected_to article_url(@article)
+  end
+
+  test "should destroy article" do
+    assert_difference("Article.count", -1) do
+      delete article_url(@article)
+    end
+
+    assert_redirected_to articles_url
+  end
+end
```

Comment:
Since we added the test, can we also add a CI for those? we can re-use the `rubocop.yml` and rename it to `ruby_ci.yml`